### ISSUE TITLE
Add proxy support for configuring ops man authentication

### ISSUE THREAD

**Issue by wrast** [pivotal-cf/pcf-pipelines#247]
Newer versions of `om` allow setting http proxy, https proxy, and no proxy. These settings are missing from the current call to `configure-authentication`.

https://github.com/pivotal-cf/pcf-pipelines/blob/5f88a5e6786da30d3fc252626d30e4162379370a/tasks/config-opsman/task.sh#L10-L16

**Comment by abbyachau** [comment]
Hi @wrast thanks for raising this. Sorry about the delay in response. Did you run into this issue at a customer site? If so, could you please get in touch about the use case? Thanks.

**Comment by wrast** [comment]
Yes this was at a customer site. We worked around it ourselves by adding the 3 additional arguments to the `om-linux` command in `task.sh`. The customer has a proxy server that is used to access all "external" sites (e.g. pivnet, github, docker hub, etc).

**Comment by abbyachau** [comment]
@wrast thanks for the information. Prioritised for engineering review

**Comment by fredwangwang** [comment]
@pivotal-maya-kenedy @calebwashburn please review and prioritize.
